android/app-example/src/main/java/com/badoo/ribs/example/rib/big/builder/BigScope.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.ribs.example.rib.big.builder
+
+import javax.inject.Scope
+
+@Scope
+@Retention(AnnotationRetention.RUNTIME)
```

Comment:
Retention [is runtime](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.annotation/-retention/index.html) by default